numpy/random/mtrand/distributions.c

```diff
@@ -500,6 +500,10 @@ long rk_poisson_mult(rk_state *state, double lam)
     }
 }
 
+/*
```

Comment:
Google shows a more complete citation.

W. Hörmann and G. Derflinger
The transformed rejection method for generating random variables, an alternative to the ratio of uniforms method
Communications in Statistics: Simulation and Computation 23(3), 847-860 (1994)